apps/spruce/vite.config.ts

```diff
@@ -16,152 +16,159 @@ import path from "path";
 import { generateBaseHTTPSViteServerConfig } from "@evg-ui/vite-utils";
 import injectVariablesInHTML from "./config/injectVariablesInHTML";
 
-const require = createRequire(import.meta.url);
+let finalConfig: Record<string, any> = {};
 
-// Remove when https://github.com/cypress-io/cypress/issues/25397 is resolved.
-dns.setDefaultResultOrder("ipv4first");
+// Only run this configuration if we are not in script mode
+if (process.env.VITE_SCRIPT_MODE !== "1") {
+  const require = createRequire(import.meta.url);
 
-// Do not apply Antd's global styles
-fs.writeFileSync(require.resolve("antd/es/style/core/global.less"), "");
-fs.writeFileSync(require.resolve("antd/lib/style/core/global.less"), "");
```

Comment:
Since this diff is way larger than it should be the only real change here is moving finalConfig into its own variable. and nesting the rest of the config under the conditional on L21